modules/react/button/lib/BaseButton.tsx

```diff
@@ -136,7 +136,7 @@ const baseButtonStyles = createStyles({
   whiteSpace: 'nowrap',
   WebkitFontSmoothing: 'antialiased',
   MozOsxFontSmoothing: 'grayscale',
-  borderRadius: cssVar(buttonVars.default.borderRadius, cssVar(system.shape.circle, '999px')),
+  borderRadius: cssVar(buttonVars.default.borderRadius, cssVar(system.shape.round, '62.5rem')),
```

Comment:
`shape.circle` is now `shape.round` and I updated the fallback value to use `rem`.